### DISCUSSION TITLE
Icons?

### DISCUSSION THREAD

**Discussion by common-47** [lepoco/wpfui#1052] [Q&A] [answered]
Where can I view the icons that are available? I have looked but cannot find them. Currently guessing what they are. Like Home24, Globe24 which seems to work. 

Is there a list?

**Comment by Lucaa2k** [comment, marked as answer]
You can find them in the Gallery App (https://apps.microsoft.com/detail/9n9lkv8r9vgm?cid=windows-lp-hero&hl=de-de&gl=DE)

Design guidance -> Icons
In Theory they are those https://github.com/microsoft/fluentui-system-icons?tab=readme-ov-file but the naming is a bit different, you should use the Gallery
